src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,12 @@
-export const UserInfo = () => {};
+// import React from 'react';
+import { User } from '../../types';
+
+interface UserInfoProps {
+  user: User | null;
+}
+
+export const UserInfo: React.FC<UserInfoProps> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user?.email}`}>
+    {user?.name}
+  </a>
+);
```

Comment:
When the user is `null` your component will render empty `a` tag - which is not ok.